phoenix-core-client/src/main/java/org/apache/phoenix/schema/PMetaDataCache.java

```diff
@@ -52,6 +53,16 @@ public void onRemoval(RemovalNotification<PTableKey, PTableRef> notification) {
                         String key = notification.getKey().toString();
                         LOGGER.debug("Expiring " + key + " because of "
                                 + notification.getCause().name());
+                        if (notification.wasEvicted()) {
+                            GlobalClientMetrics.GLOBAL_CLIENT_METADATA_CACHE_EVICTION_COUNTER.increment();
+                        } else {
+                            GlobalClientMetrics.GLOBAL_CLIENT_METADATA_CACHE_REMOVAL_COUNTER.increment();
+                        }
+                        if (notification.getValue() != null) {
+                            long newSize =
+                                    GlobalClientMetrics.GLOBAL_CLIENT_METADATA_CACHE_ESTIMATED_USED_SIZE.getMetric().getValue() - notification.getValue().getEstimatedSize();
+                            GlobalClientMetrics.GLOBAL_CLIENT_METADATA_CACHE_ESTIMATED_USED_SIZE.update(newSize);
```

Comment:
Can we do decrement here to be consistent with server side metrics computation?

Reply:
There is no `increment(long value)` method here as server side, but seems this `update(long value)` method is adding the given value to the current value. So the calculation here is wrong and we need to change to 
```
GlobalClientMetrics.GLOBAL_CLIENT_METADATA_CACHE_ESTIMATED_USED_SIZE.update(-notification.getValue().getEstimatedSize());
```
Is that okay?

Reply:
Sounds good